isis/tests/KernelDbTests.cpp

```diff
@@ -348,7 +348,7 @@ TEST_F(TestKernelDb, TestKernelsSmithOffset) {
 
   PvlGroup &dataDir = Preference::Preferences(true).findGroup("DataDirectory");
 
-  QString testDir = "data/KernelDb";
+  QString testDir = "data/KernelDB";
```

Comment:
It looks like it should be lower-case "k" here https://github.com/USGS-Astrogeology/ISIS3/tree/dev/isis/tests/data/kernelDB

Reply:
Not sure how I changed one word and still got this wrong, but thanks for the catch lol.